components/nspanel_lovelace/types.h

```diff
@@ -480,6 +486,9 @@ enum class ha_attr_type : uint8_t {
   code_arm_required,
   // cover
   current_position,
+  set_position,
```

Comment:
Please rename this from `set_position` to `position` to reflect the actual name :)

Reply:
I agree on that, see latest commit.